src/components/Sidebar/index.tsx

```diff
@@ -100,7 +123,7 @@ export default function Sidebar({ prefix, className }: { prefix: string, classNa
             defaultSelectedKeys={selectedKeys}
             openKeys={stateOpenKeys}
             onOpenChange={onOpenChange}
-            className={className}
+            className={`sidebar-container ${className || ''}`}
```

Comment:
There should be no longer a class named "sidebar-container". I think this line can retain:

```suggestion
            className={className}
```

?

Reply:
Or `className={clsx(className)}`